### DISCUSSION TITLE
how do I enable the new hydration mismatch feature?

### DISCUSSION THREAD

**Discussion by Jacubeit** [Code-Forge-Net/Remix-Dev-Tools#95] [Q&A]
I added a 
```tsx
 <form>
  <form></form>
 </form>
```
somewhere. 
Was expecting this image:
https://github.com/Code-Forge-Net/Remix-Dev-Tools#errors-tab
Image: (https://github.com/Code-Forge-Net/Remix-Dev-Tools/blob/main/assets/error-tab.png)

However, no hydration mismatch pane was shown, just an error.
Do I need to do something else?

---
remix 2.5.0
"remix-development-tools": "^3.7.2"
VSCode: Version: 1.85.1
Mac Sonoma 14.2.1

**Comment by AlemTuzlak** [comment]
@Jacubeit Any chance you could create a minimum repro? is this running on Vite or the remix bundler?
